open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/security-officer-connectors/security-officer-tag-connector/src/main/java/org/odpi/openmetadata/openconnector/governancedarmonconnectors/securityofficerconnectors/securitytagconnector/SecurityTagConnector.java

```diff
@@ -27,17 +27,32 @@ public SecurityClassification resolveSecurityClassification(SchemaElementEntity
         List<Classification> classifications = schemaElementEntity.getClassifications();
         List<Classification> existingClassifications = getExistingClassifications(schemaElementEntity, classifications);
 
+        SecurityClassification securityClassification;
         if (!existingClassifications.isEmpty()) {
-            return getMoreRestrictiveClassification(existingClassifications);
+             securityClassification = getMoreRestrictiveClassification(existingClassifications);
+
+        } else {
+            securityClassification = getDefaultSecurityTags();
+        }
+
+        Map<String, Object> properties = securityClassification.getSecurityProperties();
+        if(properties == null) {
+            properties = new HashMap<>();
         }
+        properties.put("source", schemaElementEntity.getType());
+        securityClassification.setSecurityProperties(properties);
 
-        return getDefaultSecurityTags();
+        return securityClassification;
     }
 
     private SecurityClassification getDefaultSecurityTags() {
         SecurityClassification securityClassification = new SecurityClassification();
-        securityClassification.setSecurityLabels(Collections.singletonList("C3"));
-        securityClassification.setSecurityProperties(Collections.singletonMap("user", "SecurityTagConnector"));
+        securityClassification.setSecurityLabels(Collections.singletonList("C99"));
```

Comment:
So I assume the C99 is the default security tag?

Reply:
C99 is the default security tag. The confidentiality level = 99 is "Secret", by default, if the business term does not have a Confidentiality classification assigned, schema elements are hidden.

Reply:
cool thanks!